tendrl/alerting/utils/central_store_util.py

```diff
@@ -44,3 +70,69 @@ def get_alerts():
     for alert_id, alert in alerts.iteritems():
         alerts_arr.append(AlertUtils().to_obj(alert))
     return alerts_arr
+
+
+def get_entity_alert_path(entity_type, **kwargs):
+    # TODO(Anmol Babu): Explore the possibility of getting the below info
+    # from definitions instead of hard coding here and if that works
+    # Implement it in all other above functions where etcd path is hardcoded.
+    entity_alert_path_map = {
+        alerting_consts.CLUSTER: '/alerting/clusters/$cluster_id',
+        alerting_consts.NODE: '/alerting/nodes/$node_id'
+    }
+    return Template(
+        entity_alert_path_map.get(
+            entity_type,
+            alerting_consts.NOT_AVAILABLE
+        )
+    ).substitute(kwargs)
+
+
+def get_entity_alerts(entity_type, **kwargs):
+    alerts = []
+    try:
+        alert_path = get_entity_alert_path(entity_type, **kwargs)
+        alerts = read(
+            alert_path
+        )
+        for alert_id, alert in alerts.iteritems():
+            alerts.append(alert)
+        return alerts
+    except Exception:
+        return alerts
+
+
+def get_alert_counter_update_path(entity_type, **kwargs):
+    entity_alert_counter_path = {
+        alerting_consts.CLUSTER: '/clusters/$cluster_id',
+        alerting_consts.NODE: '/nodes/$node_id'
+    }
+    return Template(
+        entity_alert_counter_path.get(
+            entity_type,
+            alerting_consts.NOT_AVAILABLE
+        )
+    ).substitute(kwargs)
+
+
+def update_alert_counters(
+    entity_type,
+    crit_alert_count,
+    warn_alert_count,
+    **kwargs
+):
+    alert_counter_path = get_alert_counter_update_path(entity_type, **kwargs)
+    NS._int.client.write(
```

Comment:
Please make objects out of alert counters, we need to stop doing direct write to etcd in new PRs, the object.save() handles a lot of etcd exceptions which arent handled here